pkg/logs/epforwarder/epforwarder.go

```diff
@@ -0,0 +1,158 @@
+package epforwarder
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/auditor"
+	"github.com/DataDog/datadog-agent/pkg/logs/client"
+	"github.com/DataDog/datadog-agent/pkg/logs/client/http"
+	"github.com/DataDog/datadog-agent/pkg/logs/config"
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
+	"github.com/DataDog/datadog-agent/pkg/logs/restart"
+	"github.com/DataDog/datadog-agent/pkg/logs/sender"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	eventTypeDBMSamples = "dbm-samples"
+	eventTypeDBMMetrics = "dbm-metrics"
```

Comment:
Should we expose those as constant to the Python checks ?

Also why are we working with strings and not integers (aka const + iota) ?

Reply:
> Should we expose those as constant to the Python checks ?

We could, but I'm not sure that would be help much. For now we're planning on having a dedicated aggregator submit method per event type, so the python checks won't even be calling this directly. See https://github.com/DataDog/integrations-core/pull/9045/files#diff-5a536a3141763ea2d55833f492361fc4c36e9de806350a2dd24945b149e10505R520 

If we proceed with that approach then I'm not sure that exposing constants is going to help much as they'll be used only in the one place in python code. 

> Also why are we working with strings and not integers (aka const + iota) ?

The main reason I figured is that it's a human readable token that can easily be grepped across agent and integrations code, as well as be used directly as a tag in the aggregator expvars & telemetry. 

If we use const + iota then we'll need some way to communicate that iota value to the python code and we'll have to add add translation logic to/from that iota and tag values for telemetry. So I think using const + iota would make things more complicated with not much benefit. 

Can you think of a good reason why that would be better to use here?

Reply:
OK make sens. I think once we had more event type we can re-evaluate exposing constant to the Python code.